ynr/apps/official_documents/models.py

```diff
@@ -25,6 +25,7 @@ def document_file_name(instance, filename):
 
 @python_2_unicode_compatible
 class OfficialDocument(TimeStampedModel):
+    # TODO FK to post_election and remove the Election and Post FKs
```

Comment:
There are a couple of TODOs in this PR (there's another in `add_person()` ). Once we've agreed on something to merge, can we raise issues for anything outstanding as appropriate so they aren't forgotten or ignored.

Reply:
That comment isn't touched in this PR I don't think. In general there are a lot of TODOs in the code – it's a good point about turning them in to issues.

I reckon all the TODOs in this PR are gone, mainly because, well, I did them.